2.10/2.10.24/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.10.24
+Человек вводит время, чтобы поставить будильник.
+Выводится сообщение для ввода часов: "Введите часы"
+Потом для ввода минут: "Введите минуты"
+Есть условия у этого будильника:
+- значения часов может быть 0 - 23
+- значения минут может быть 0 - 59
+- будильник можно ставить только утром с 5:00 до 8:00 включительно
+
+После ввода всех данных: проверить все условия (ифы не вкладывать друг в друга)
+И если всё ок, тогда вывести алерт "Будильник готов", иначе "Неправильные данные"
+<script>
+	let hasError = false;
+	const hours = +prompt('Введите часы');
+	let minutes = +prompt('Введите минуты');
```

Comment:
let or const?

and it's more correct to declare variables at first `const` and then `let`:
```
const var1 = ...;
const var2 = ...;

let var3 = ...;
let var4 = ...;
```